packages/migration/src/migrations/user-mgnt/20231106130956-email-mobile-unique-index.ts

```diff
@@ -15,6 +15,40 @@ export const up = async (db: Db, client: MongoClient) => {
   const session = client.startSession()
   try {
     await session.withTransaction(async () => {
+      console.log('Checking if mobile or email field already have indices')
+      const indexes = Object.keys(
+        (await db.collection('users').stats()).indexSizes
+      )
+      const mobileHasIndex = indexes.some(
+        (index) => index.split('_')[0] === ('mobile' as const)
+      )
+      const emailForNotificationHasIndex = indexes.some(
+        (index) => index.split('_')[0] === ('emailForNotification' as const)
+      )
+
+      if (mobileHasIndex) {
+        console.log('Removing index of mobile')
+        await db.collection('users').dropIndex({ mobile: 1 } as any)
```

Comment:
```suggestion
        await db.collection('users').dropIndex('mobile')
```

I think this is what the typing wants

Reply:
```
rs0:PRIMARY> db.users.getIndexes()
[
	{
		"v" : 2,
		"key" : {
			"_id" : 1
		},
		"name" : "_id_"
	},
	{
		"v" : 2,
		"unique" : true,
		"key" : {
			"emailForNotification" : 1
		},
		"name" : "emailForNotification_1",
		"sparse" : true
	},
	{
		"v" : 2,
		"unique" : true,
		"key" : {
			"mobile" : 1
		},
		"name" : "mobile_1",
		"sparse" : true
	}
]
```
the index name seems to be `mobile_1`

Reply:
You most likely can get the right index name from you code above 

```ts
      const mobileHasIndex = indexes.some(
        (index) => index.split('_')[0] === ('mobile' as const)
      )
```

if you change that to a find operation